src/app.rs

```diff
@@ -455,3 +409,48 @@ async fn sources_finished(mutex: Arc<Mutex<TopologyController>>) {
         }
     }
 }
+
+fn get_log_levels(default: &str) -> String {
+    std::env::var("VECTOR_LOG")
+        .or_else(|_| {
+            warn!(message = "Use of $LOG is deprecated. Please use $VECTOR_LOG instead.");
+            std::env::var("LOG")
+        })
```

Comment:
```suggestion
        .or_else(|_| {
            warn!(message = "DEPRECATED: Use of $LOG is deprecated. Please use $VECTOR_LOG instead.");
            std::env::var("LOG")
        })
```

I know this was kept as-is but can we correct this message (per the deprecation docs). Also throwing a reminder into the void that we should remove this sooner than later, it's been a few releases.

Reply:
Will do. Looking at it, that code would also suggest we _always_ print that warning if `$VECTOR_LOG` is unset.

Reply:
🤔 I'd also think so.... but I don't think I see that log normally, and I'm usually not setting anything.